src/SqlPersistence/SerializationException.cs

```diff
@@ -7,5 +7,8 @@ class SerializationException : Exception
         public SerializationException(Exception innerException) : base("Serialization failed", innerException)
         {
         }
+        public SerializationException(string message) : base(message)
```

Comment:
This could go when the code would not throw a serialization exception for the misconfiguration